hypershift-operator/controllers/hostedcluster/karpenter.go

```diff
@@ -105,7 +106,14 @@ spec:
 	// TODO(alberto): Ensure deletion if autoNode is disabled.
 
 	// Run karpenter Operator to manage CRs management and guest side.
-	if err := karpenteroperatormanifest.ReconcileKarpenterOperator(ctx, createOrUpdate, r.Client, hypershiftOperatorImage, controlPlaneOperatorImage, hcp); err != nil {
+
+	// TODO(jkyros): Grab the karpenter image in the proper place at the beginning with args, not here?
+	karpenterProviderAWSImage, hasImage := releaseImage.ComponentImages()["aws-karpenter-provider-aws"]
```

Comment:
will this be the name for this image in the payload, aws -*-aws seems redudant

Reply:
Yes, that was the planned name for the payload. 

I don't disagree, but it's the name of our forked downstream repo,  and it matches the naming standard of "cloud-provider specific names being prefixed with their cloud provider" from https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md#segmentation-ordering. 

In this case it unfortunately just so happens that the name also _ends_ with the cloud provider too :/